setup.bash

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
Not sure, I just used `.bash` as a convention I picked up from others I work with. 
There is a [third option](https://stackoverflow.com/a/27824204) of no extension at all, but personally I like the verbosity of `.bash`.

Reply:
Interesting. I thought the extension was important for syntax highlighting, but vscode can look at the shebang to decipher syntax. Not sure about GitHub.

The extension is nice IMO because it:

1. makes it easier to filter for bash scripts based on filename
2. makes it clear it needs to be used with bash (and we're not executing the file directly)
3. allows for a directory without the extension.

I am okay with keeping .bash and then switching our other scripts to use .bash (if they require bash).